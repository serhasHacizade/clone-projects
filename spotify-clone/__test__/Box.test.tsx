import React from 'react';
import { render } from '@testing-library/react';
import Box from '../components/Box'; // YourComponent dosyasını buraya doğru şekilde import etmelisiniz

test('YourComponent renders with correct styles and children', () => {
  const mockChildren = 'Test Children';

  const { container, getByText } = render(
    <Box className="test-class">
      {mockChildren}
    </Box>
  );

  // Componentin doğru şekilde render edildiğini kontrol ediyoruz
  expect(getByText('Test Children')).toBeTruthy()

});
