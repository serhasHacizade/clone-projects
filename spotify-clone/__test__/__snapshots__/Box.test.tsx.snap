// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`YourComponent renders with correct styles and children 1`] = `
<div
  class="test-class"
>
  Test Children
</div>
`;
